iOS/Flows/Room/RoomViewController.swift

```diff
@@ -114,7 +114,9 @@ class RoomViewController: DiffableCollectionViewController<RoomSectionType,
         var data: [RoomSectionType: [RoomItemType]] = [:]
         
         try? await NoticeStore.shared.initializeIfNeeded()
-        let notices = await NoticeStore.shared.getAllNotices()
+        let notices = await NoticeStore.shared.getAllNotices().filter({ notice in
```

Comment:
We will display individual notices for each unread message the user has so no need to show this one.